### PR TITLE
remove LSP support

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,29 +62,20 @@ This should create a tab in your browser with the development app that contains
 
 ### Autocompletion
 
-The autocompletion feature has 3 different modes, each requiring a different setup:
+The autocompletion feature has 2 different modes, each requiring a different setup:
```
Comment (dvkndn) [inline README.md:65]: I'm really like this change (it makes the setup much easier) but I'm still curious... is there a place I can read more about the reason behind this?

Comment (Nexucis) [reply README.md:65]: yeah it's because of this: https://github.com/prometheus/prometheus/pull/6872#issuecomment-703505588

Comment (dvkndn) [reply README.md:65]: Thank you!

**Comment by juliusv** [comment]
:+1:

**Pull request merged**